src/ScoreBoard.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import "./scoreboard.css";
+
+/* 
+    Read saved user moves from localStorage
+    Render Scoreboard
+    
+    ** Contributed by Soham Mondal <contact@sohammondal.com> **
+  */
+
+class Scoreboard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      userMoves: null
+    };
+  }
+
+  componentDidMount() {
+    this.getUserMovesFromLocalStorage();
+  }
+  getUserMovesFromLocalStorage() {
+    let savedUserMoves = localStorage.getItem("userMoves");
+    if (savedUserMoves) {
+      savedUserMoves = JSON.parse(savedUserMoves);
+      savedUserMoves.sort((user1, user2) => user1.moves - user2.moves);
+      this.setState({ userMoves: savedUserMoves });
+    }
+  }
+
+  renderScoreboard(userMovesArray) {
```

Comment:
instead of passing state data to render function via params you can just use the state in the function itself.

Something like

```js
renderScoreboard() {
  const { userMoves } = this.state;
  // now user userMoves instead of userMovesArray.
}
```